MVVM Module/Source/Feed Presentation/FeedViewModel.swift

```
@@ -17,16 +17,22 @@ final class FeedViewModel {
 	var title: String {
         Localized.Feed.title
 	}
+    
+    var errorMessage = Localized.Feed.loadError
 
 	var onLoadingStateChange: Observer<Bool>?
 	var onFeedLoad: Observer<[FeedImage]>?
+    var onFeedFailed: Observer<String>?
```

As you did in the MVP solution, the error message could be an `Optional<String>` - so the ViewModel can send `.none` when the UI should hide the error. You can rename it as well to make this behavior clear:

```
var onErrorStateChange: Observer<String?>?
```